pages/common/dlv.md

```diff
@@ -0,0 +1,24 @@
+# dlv
+
+> Delve is a debugger for the Go programming language.
+> More information: <https://github.com/go-delve/delve/blob/master/Documentation/usage/dlv.md/>.
+
+- Compile and begin debugging main package in current directory, or the package specified:
+
+`dlv debug`
+
+- Compile test binary and begin debugging program:
+
+`dlv test`
+
+- Connect to a headless debug server:
+
+`dlv connect`
+
+- Attach to running process and begin debugging:
+
+`dlv attach`
+
+- Compile and begin tracing program:
+
+`dlv trace`
```

Comment:
Looks like this needs more info: `dlv trace [package] regexp [flags]`

Reply:
Thanks. I was wondering if this would work (I read the style guide but need to doublecheck): ```'dlv trace {{package}} --regexp {{flags}}```